src/main/kotlin/no/nav/familie/tilbake/behandling/domain/Behandling.kt

```diff
@@ -42,7 +43,8 @@ data class Behandling(
     @Version
     val versjon: Long = 0,
     @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
-    val sporbar: Sporbar = Sporbar()
+    val sporbar: Sporbar = Sporbar(),
+    val regelverk: Regelverk? = null
```

Comment:
Default verdi bør kanskje være Nasjonal?
Så vi har Regelverk også på eksisterande/historiske lagra behandlingar?

Reply:
Ja, vurderte det litt, men veit ikkje om det blir rett? Eller, vi har kanskje ikkje starta med tilbakekreving for EØS enno?

Reply:
Eivind fann i forrige veke ut at det iallefall var 3 tilbakekrevinger opprettet som burde ha hatt EØS regelverk. Eg gjorde eit nytt søk, og det var nok fleire.
Men tror ingen av dei vil bli tatt ut for automatisk vurdering, siden beløpa er for store. Eg tror det er trygt å sette Nasjonal som default. Tror det er betre enn at me ikkje har noko regelverk satt

Reply:
Erfaringa mi frå andre stadar er at det er betre å ikkje setje noko enn å setje noko som potensielt er feil. Men er ikkje stø nok i konteksten vår her enno til å vera bastant på det.

Kva er konsekvensen om vi feilkategoriserer her? Har saksbehandlar moglegheit til å rette det opp?

Reply:
Konsekvensen er liten. men nei saksbehandler har ikkje mulighet til å rette det opp.
Kan la det være slik, også eventuelle legge på default verdi ved eit seinare tidspunkt.